dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WsAddressStreamRealTimeTestSuite.scala

```diff
@@ -0,0 +1,59 @@
+package com.wavesplatform.it.sync.api.ws
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.dex.api.ws.connection.WsConnection
+import com.wavesplatform.dex.api.ws.entities.{WsBalances, WsMatchTransactionInfo, WsOrder}
+import com.wavesplatform.dex.domain.account.KeyPair
+import com.wavesplatform.dex.domain.account.KeyPair.toAddress
+import com.wavesplatform.dex.domain.asset.Asset.Waves
+import com.wavesplatform.dex.domain.order.OrderType.{BUY, SELL}
+import com.wavesplatform.dex.model.OrderStatus
+import com.wavesplatform.it.WsSuiteBase
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+import scala.util.Using
+
+class WsAddressStreamRealTimeTestSuite extends WsSuiteBase with TableDrivenPropertyChecks {
```

Comment:
I think we need to set at least tick in 1hour to be sure that we can receive an update independently of it in eventually (within ~2 minutes).

Also I don't see where we need `TableDrivenPropertyChecks`) And check file for unused imports

Reply:
removed TableDrivenPropertyChecks

Reply:
and removed ubused imports